kolibri/plugins/audio_mp3_render/webpack.config.js

```diff
@@ -1,16 +0,0 @@
-/*
- * This file defines additional webpack configuration for this plugin.
- * It will be bundled into the webpack configuration at build time.
- */
-
-module.exports = {
-  module: {
-    loaders: [
-      // Allows <video> and <audio> HTML5 tags work on all major browsers.
-      {
-        test: require.resolve('html5media/dist/api/1.1.8/html5media'),
-        loader: "imports?this=>window"
-      }
-    ]
-  }
-};
```

Comment:
It's weird to keep an unused file. I will add a TODO comment for the inline loader.